app/views/search.nunjucks

```diff
@@ -15,6 +15,9 @@
         <!-- label -->
       </label>
 
+      {% if errorMessage %}
```

Comment:
Good spot, I've fixed it of sorts. The input will not be reloaded but getting rid of the error message is another thing altogether.
Notes on how WebKit caches pages https://webkit.org/blog/427/webkit-page-cache-i-the-basics/